PeerioTechnologies/peerio-desktop
# fix: disabled MenuItem click through

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/7027/desktop-disabled-menu-items-are-click-through
 
#### Testing instructions  
Disabled MenuItem should no longer click through.

This doesn't come up on `dev` at all so to test you'll have to fake it to repro the bug.

* Go to src/ui/chat/components/sidebar/FilesSection.jsx
* Find the "Share" MenuItem and add "disabled" property to it
 ```
<MenuItem
  caption={t('button_share')}
  icon="reply"
  onClick={this.share}
  disabled  // add this line
/>
```
* `npm start` a desktop instance
* Open a chat that has files in it
  * Open the sidebar.
  * Press the "three dots" menu on any file. "Share" should be disabled
  * Clicking on "Share" should *not* activate the download event (This is what was happening on `newfs`). It should just make the Menu fade out.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
